packages/frontend/src/hooks/echarts/useEchartsCartesianConfig.ts

```diff
@@ -419,7 +399,8 @@ const getMinAndMaxReferenceLines = (
             if (serieFieldId.field !== fieldId) return [];
 
             if (!serie.markLine) return [];
-            const field = findItem(items, serieFieldId.field);
+            // TODO: check if correct because findItem can check for name
```

Comment:
remind todo

Reply:
Pivot doesn't work -> https://lightdash-pr-8168.onrender.com/share/wtZjg2qJ4Odb_87Ua_rsr

Reply:
todo: done and checked, this is handled by the `getItemMap` so all good

on pivoting, the fix is here @ZeRego : https://github.com/lightdash/lightdash/pull/8168/commits/8bbd52e867a03a9fb51c07089929be25b247b16e 

Waiting as well for e2e tests to see what else might have been affected